packages/broker/test/unit/ConfigWizard.test.ts

```diff
@@ -6,14 +6,11 @@ import { PROMPTS, DEFAULT_CONFIG_PORTS, selectStoragePathPrompt, createStorageFi
 
 const assertValidPort = (port: number | string, pluginName = 'websocket') => {
     const numericPort = (typeof port === 'string') ? parseInt(port) : port
-    const privateKeyAnswers = {
-        generateOrImportPrivateKey: 'Generate',
-    }
     const pluginsAnswers = {
         selectPlugins:[pluginName],
         websocketPort: port,
     }
-    const privateKey = getPrivateKey(privateKeyAnswers)
```

Comment:
When we validate the port in assertValidPort, we don't need any info about the port. Would 
`const config = getConfig(undefined as any, pluginsAnswers)` be a minimalistic valid way to get the port configs?